lib/jpegstream.js

```diff
@@ -10,7 +10,8 @@
  * Module dependencies.
  */
 
-var Stream = require('stream').Stream;
+var Readable = require('stream').Readable;
```

Comment:
You should depend from `readable-stream`, it is better practice and yields more stability across node versions.